cmd/blocking-issue-creator/main.go

```diff
@@ -136,6 +132,17 @@ func manageIssues(client githubClient, githubLogin string, repoInfo *config.Info
 		return err
 	}
 
+	if len(branches) == 0 {
+		for _, issue := range issues {
+			if err := client.CloseIssue(repoInfo.Org, repoInfo.Repo, issue.Number); err != nil {
+				logger.WithError(err).Error("Failed to close issue.")
+				return err
+			}
+			logger.WithField("number", issue.Number).Info("Closed issue.")
+		}
```

Comment:
suggestion: maybe extract this into a `closeIssues(issues []Issue, repoInfo, logger) error` and use it both here and in the `len(issues) > 1` block below.

Could be even an anonymous funciton inline to simplify the signature :

```go
closeIssues := func(issues []Issue) error {
}
```

Reply:
Actually, I thought about this redundancy for a little while and came to the conclusion it does not make much sense in this situation, but I can adjust this.

Reply:
My rationale was: logs are different and we are acting on different sets: issues from 0 vs issues from 1

Reply:
Yeah I thought it would take a slice as an input, and then it could be called as `closeIssues(issues)` from one location and `closeIssues(issues[1:])` from the other. But not a big deal.